content/bitcoin-core.md

```diff
@@ -13,17 +13,16 @@ Using an ASmap file in one's Bitcoin Core configuration is already strictly bett
 
 ## Usage
 
-#### Create an ASmap with Kartograf
-
-[Kartograf](https://github.com/fjahr/kartograf) is a tool to fetch AS data from regional IRRs, and produce a file to be used with Core. Once we have a reliable process for publishing ASmaps, we'll make them available for public download (tbd).
+`bitcoin` will accept a compressed ASmap file with the `-asmap` option.
+You can download a pre-made [latest_asmap.data](https://github.com/fjahr/asmap-data/blob/main/latest_asmap.dat) file in the [asmap-data](https://github.com/fjahr/asmap-data) repo.
 
-#### Compress it with `asmap-tool` in Core
+#### Create an ASmap with Kartograf
 
-[asmap-tool](https://github.com/bitcoin/bitcoin/tree/master/contrib/asmap) is a Python script to help encode/compress an ASmap file before passing it to Core.
+You can choose to generate an ASmap file yourself. [Kartograf](https://github.com/fjahr/kartograf) is a tool to fetch AS data from regional IRRs, and produce a file to be used with Bitcoin Core.
 
-#### Pass the ASmap to Core
+#### Compress it with `asmap-tool`
 
-Core will accept a compressed ASmap file with the `-asmap` option as described [here](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.20.0.md#new-settings).
+If you generate a file yourself, you must compress it before passing it to `bitcoind`. [asmap-tool](https://github.com/bitcoin/bitcoin/tree/master/contrib/asmap) is a Python script to help encode/compress an ASmap file which ships with Bitcoin Core.
```

Comment:
This is really nit picking but 'ships with' sounds to me like it would be usable for someone who downloads a release. But you really need to get the repository.